include/native/windows/windows_base.h

```diff
@@ -35,7 +38,7 @@ typedef uint16_t UINT16;
 typedef uint32_t UINT32;
 typedef uint64_t UINT64;
 typedef void VOID;
-typedef void* LPVOID;
+typedef void* PVOID, LPVOID;
```

Comment:
This is wrong, it changes `LPVOID` to non-pointer.

Reply:
Sorry, my bad. Fixed.